File: website/random/hello.py
#!/usr/bin/env python3

import cgi
import sys

# print("Content-Type: text/html\n")
sys.stdout.write("Content-Length: 144\r\n")
sys.stdout.write("Content-Type: text/html\r\n\r\n")

print("<html>")
print("<head>")
print("<title>Simple Python CGI</title>")
print("</head>")
print("<body>")
print("<h1>Hello, World!</h1>")
print("<p>This is a simple Python CGI script.</p>")
print("</body>")
print("</html>")

File: website/first/hello.php
#!/usr/bin/php-cgi
<?php
header("Content-Type: text/html");

echo "<!DOCTYPE html>";
echo "<html lang='en'>";
echo "<head>";
echo "<meta charset='UTF-8'>";
echo "<title>Hello World CGI</title>";
echo "</head>";
echo "<body>";
echo "<h1>Hello World from PHP CGI!</h1>";
echo "<p>Current date and time: " . date('Y-m-d H:i:s') . "</p>";
// echo "<p>Server name: " . $_SERVER['SERVER_NAME'] . "</p>";
// echo "<p>CGI version: " . $_SERVER['GATEWAY_INTERFACE'] . "</p>";
// echo "<p>Client IP: " . $_SERVER['REMOTE_ADDR'] . "</p>";
echo "</body>";
echo "</html>";
?>